app/org/sagebionetworks/bridge/validators/StudyParticipantValidator.java

```diff
@@ -0,0 +1,87 @@
+package org.sagebionetworks.bridge.validators;
+
+import java.util.Set;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.validator.routines.EmailValidator;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import org.sagebionetworks.bridge.BridgeUtils;
+import org.sagebionetworks.bridge.models.accounts.StudyParticipant;
+import org.sagebionetworks.bridge.models.studies.PasswordPolicy;
+import org.sagebionetworks.bridge.models.studies.Study;
+
+public class StudyParticipantValidator implements Validator {
+
+    private final EmailValidator emailValidator = EmailValidator.getInstance();
+    private final Study study;
+    private final boolean isNew;
+    
+    public StudyParticipantValidator(Study study, boolean isNew) {
+        this.study = study;
+        this.isNew = isNew;
+    }
+    
+    @Override
+    public boolean supports(Class<?> clazz) {
+        return StudyParticipant.class.isAssignableFrom(clazz);
+    }
+
+    @Override
+    public void validate(Object object, Errors errors) {
+        StudyParticipant participant = (StudyParticipant)object;
+        
+        if (isNew) {
```

Comment:
The isNew flag here seems to answer my question about the differences between createParticipant and updateParticipant are. Could we structure the service like this method? Here, it's a lot clearly what's going on and why